### PR TITLE
[PS-686] Mobile update negative copy in settings

### PR DESCRIPTION
**Pull request by coroiu**
## Type of change
- [ ] Bug fix
- [ ] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [x] Other

## Objective

Update Negative Copy in Settings

All clients do not use a consistent method for labeling settings controls. We often see settings that include the word "Disable" along with a checkmark which implies "enabling" a disable; this can be [confusing for users](https://www.reddit.com/r/Bitwarden/comments/qhns9l/browser_extensions_can_now_autoprompt_for/).

## Code changes

* **file.ext:** Description of what was changed and why

## Before you submit
- [x] I have checked for formatting errors (`dotnet tool run dotnet-format --check`) (required)
- [ ] I have added **unit tests** where it makes sense to do so (encouraged but not required)
- [ ] This change requires a **documentation update** (notify the documentation team)
- [ ] This change has particular **deployment requirements** (notify the DevOps team)

### CODE REVIEW

File: src/Android/Properties/AndroidManifest.xml [hunk 1]
```diff
@@ -1,57 +1,48 @@
-<?xml version='1.0' encoding='UTF-8'?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (mp-bw) [inline src/Android/Properties/AndroidManifest.xml:1]: Visual Studio for Mac has a weird issue where simply opening the mobile project completely (and improperly) re-formats the Android manifest.  Unless there are required changes, the entire file should be reverted for the PR.

**Review by fedemkr — CHANGES_REQUESTED** [review]
Nice work! Just a few things with the string resources.

File: src/Android/Properties/AndroidManifest.xml (hunk 1 above)
Comment (coroiu) [reply src/Android/Properties/AndroidManifest.xml:1]: Oh, that's.. interesting :)
I'll fix!

File: src/App/Resources/AppResources.resx [hunk 2]
```diff
@@ -907,11 +907,11 @@
...
-  <data name="DisableAutoTotpCopyDescription" xml:space="preserve">
-    <value>If your login has an authenticator key attached to it, the TOTP verification code is automatically copied to your clipboard whenever you auto-fill the login.</value>
+  <data name="AutoTotpCopyDescription" xml:space="preserve">
+    <value>If a login has an authenticator key, copy the TOTP verification code to your clip-board when you auto-fill the login.</value>
   </data>
-  <data name="DisableAutoTotpCopy" xml:space="preserve">
-    <value>Disable Automatic TOTP Copy</value>
+  <data name="AutoTotpCopy" xml:space="preserve">
+    <value>Copy TOTP automatically</value>
   </data>
```
[the remark this answers is not in the document]
Comment (coroiu) [reply src/App/Resources/AppResources.resx]: Updated the keys to reflect the content! 
The function names and variables (incl. state service) also need to be changed, but I've added a tech debt ticket for that [PS-961]

**Comment by fedemkr** [comment]
@coroiu shall this be merged now or is this on hold for something?

**Pull request merged**